lib/es6/Lexer.js

```diff
@@ -0,0 +1,107 @@
+// A simple lexer for SQL.
```

Comment:
@dougwilson This separate file is now imported by the unittest instead getting rid of the conditional export.